Bottom: 405b53310e6dcbd971bc7aacad4f5d976c385266
Top:    b04143adafcfea492c642b069b4a5317fce73a5c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-14 20:03:32 +0000

Refresh of tmp-fixes

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index e2d673a..6339408 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -997,6 +997,7 @@ ps_set_layer (const char *name, int group, int empty)
      and we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
       strcmp (name, "outline") != 0 &&
